apps/teams-test-app/e2e-test-data/otherAppStateChange.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "OtherAppStateChange",
+  "platforms": "Web",
+  "hostSdkVersion": {
+    "web": ">2.13.1"
+  },
+  "testCases": [
+    {
+      "version": ">2.21.0",
+      "title": "isSupported - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_otherAppStateChange_isSupported",
+      "expectedTestAppValue": "OtherAppStateChanged module is supported"
+    },
+    {
+      "version": ">2.21.0",
+      "title": "registerAppInstallationHandler - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_otherAppStateChange_registerInstallHandler",
+      "expectedTestAppValue": "received"
+    },
+    {
+      "version": ">2.21.0",
+      "title": "registerAppInstallationHandler should not receive install events if app is not approved",
+      "type": "registerAndRaiseEvent",
+      "boxSelector": "#box_otherAppStateChange_registerInstallHandler",
+      "eventName": "otherApp.install",
+      "eventData": [
+        {
+          "appIds": ["123", "456"]
+        }
+      ],
+      "expectedTestAppValue": "received"
+    },
+    {
+      "version": ">2.21.0",
```

Comment:
Forget to mention, if you want, you could have version over the file as a capability-wide field to control test cases.